frontend/integration-tests/tests/login.scenario.ts

```diff
@@ -39,6 +39,31 @@ describe('Auth test', () => {
         expect(loginView.userDropdown.getText()).toContain(BRIDGE_HTPASSWD_USERNAME);
       });
 
+      it('does not show admin nav items in Administration to htpasswd user', async() => {
+        await browser.wait(until.visibilityOf(sidenavView.navSectionFor('Administration')));
+        expect(sidenavView.navSectionFor('Administration')).not.toContain('Cluster Status');
+        expect(sidenavView.navSectionFor('Administration')).not.toContain('Cluster Settings');
+        expect(sidenavView.navSectionFor('Administration')).not.toContain('Namespaces');
+        expect(sidenavView.navSectionFor('Administration')).not.toContain('Custom Resource Definitions');
+      });
+
+      it('does not show admin nav items in Catalog to htpasswd user', async() => {
+        await browser.wait(until.visibilityOf(sidenavView.navSectionFor('Catalog')));
+        expect(sidenavView.navSectionFor('Catalog')).not.toContain('OperatorHub');
+        expect(sidenavView.navSectionFor('Catalog')).not.toContain('Operator Management');
```

Comment:
We are reenabling this menu in #1545

Reply:
Looks like it merged. Tests are passing now.

Reply:
@rebeccaalpert This nav section *should* be visible, so we should just remove this line.